plugins/modules/rds_instance.py

```diff
@@ -982,8 +1052,13 @@ def get_changing_options_with_inconsistent_keys(modify_params, instance, purge_c
                 if desired_option in current_option:
                     continue
 
-        if current_option == desired_option:
-            continue
+        if option != 'ProcessorFeatures':
```

Comment:
Nitpicking and please don't refactor your code just because of my feeling. I think these two blocks are a bit hard to read, I would flatten the if/else this way.

```python
if option != 'ProcessorFeatures' and current_option == desired_option:
    continue

if option == 'ProcessorFeatures' and current_option == boto3_tag_list_to_ansible_dict(desired_option, 'Name', 'Value'):
    continue
```